### PR TITLE
chore: update more deps

### PR DESCRIPTION
**Pull request by lzap**
TIL: `go get -u all` is not the same as `go get -u ./...` and the latter is what we want to use. This patch fixes that, I am not entirely what "all" exactly does but it did not work.

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -236,8 +236,8 @@ func Initialize(configFiles ...string) {
 					config.Kafka.CACert = *b.Cacert
 				}
 				if b.Sasl != nil {
-					if b.Sasl.SecurityProtocol != nil && *b.Sasl.SecurityProtocol != "" {
-						config.Kafka.SASL.SecurityProtocol = *b.Sasl.SecurityProtocol
+					if b.SecurityProtocol != nil && *b.SecurityProtocol != "" {
+						config.Kafka.SASL.SecurityProtocol = *b.SecurityProtocol
```
Comment (lzap) [inline internal/config/config.go:240]: This was deprecated in clowder and moved up one level, linter errors on deprecations.

File: internal/telemetry/otel.go [hunk 2]
```diff
@@ -52,17 +52,11 @@ func Initialize(rootLogger *zerolog.Logger) *Telemetry {
...
-			semconv.ServiceVersionKey.String(version.OpenTelemetryVersion),
-		),
+	res := resource.NewWithAttributes(
+		semconv.SchemaURL,
+		semconv.ServiceNameKey.String("provisioning"),
+		semconv.ServiceVersionKey.String(version.OpenTelemetryVersion),
 	)
-	if err != nil {
-		panic(err)
-	}
```
Comment (lzap) [inline internal/telemetry/otel.go:65]: The new OpenTelemetry panics with schema merge problem, this is how you do it now.

**Comment by lzap** [comment]
Fixed tests.

**Comment by lzap** [comment]
Tests are passing!

**Review by ezr-ondrej — APPROVED** [review]
:exploding_head: no idea what's going on with `all` vs `./...`, but looks reasonable :shrug: :)

**Comment by lzap** [comment]
Even folks on the Go Slack channel were unable to tell, the conclusion was: use `./...`

**Pull request merged**